README-DEV.rst

```diff
@@ -317,6 +317,32 @@ used to configure Clouseau to work with that::
 
     dev/run --with-clouseau --erlang-cookie=brumbrum
 
+To facilitate a smoother development flow, it is possible to override
```

Comment:
Wordy description that isn't imo useful to the reader. Can we tighten the text here to what these options do and why they might be useful? Striking everything before "it" in the first sentence is the kind of edit I'm thinking of.

Reply:
Sure, I am fine with making it more to the point.